app/models/public_body.rb

```diff
@@ -68,7 +68,9 @@ class ImportCSVDryRun < StandardError ; end
   validates :request_email,
             not_nil: { message: N_("Request email can't be nil") }
 
-  validates_uniqueness_of :short_name, :message => N_("Short name is already taken")#, :allow_blank => true
+  validates_uniqueness_of :short_name,
+                          :message => N_("Short name is already taken"),
+                          :allow_blank => true
```

Comment:
This (and the spec removal) should get squashed in to https://github.com/mysociety/alaveteli/pull/4084/commits/d8268fda2b5603a98351582020d6b5df7d787d83